insights/tests/parsers/test_nginx_log.py

```diff
@@ -1,6 +1,6 @@
 from insights.tests import context_wrap
 from insights.parsers import nginx_log
-from insights.parsers.nginx_log import NginxErrorLog
+from insights.parsers.nginx_log import NginxErrorLog, ContainerNginxErrorLog
```

Comment:
Could you please move ContainerNginxErrorLog before NginxErrorLog so it's in alphabetical order?

Reply:
@ryan-blakley - Updated, please check